app/lib/tasksManager.js

```diff
@@ -140,6 +144,37 @@ export default class TasksManager {
     this.tasks = this.tasks.move(from, to);
   }
 
+  sortTasks(sort) {
```

Comment:
I would use a boolean for sort rather than a string. You could also use enums. Something like:

```
const SortType = {
   Alphabetical: 1,
   Reverse: 2
}
```

This really just creates an object with the key is Alphabetical and the value is 1. But it makes it less likely you'll make a mistake typing in a string.